taxi/migrations/0003_rename_manufactured_car_manufacturer.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.2 on 2024-01-26 00:39
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('taxi', '0002_manufacturer_car'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='car',
+            old_name='manufactured',
+            new_name='manufacturer',
+        ),
+    ]
```

Comment:
It is a good practice to include one migration per app per pull request in Django projects to not overload the `migrations` folder. Include one migration to this pull request, please.